cmake/toolchains/ARM.cmake

```diff
@@ -35,9 +40,9 @@ target_compile_options(mbed-os
 
 target_compile_options(mbed-os
     PUBLIC
-        $<$<COMPILE_LANGUAGE:ASM>:${mbed_studio_arm_compiler}>
+        $<$<COMPILE_LANGUAGE:ASM>:${MBED_STUDIO_ARM_COMPILER}>
         $<$<COMPILE_LANGUAGE:ASM>:--cpreproc>
-        $<$<COMPILE_LANGUAGE:ASM>:--cpreproc_opts=--target=arm-arm-none-eabi,-mcpu=cortex-m4,-D,__FPU_PRESENT,-D,MBED_CONF_PLATFORM_CRASH_CAPTURE_ENABLED>
+        $<$<COMPILE_LANGUAGE:ASM>:--cpreproc_opts=--target=arm-arm-none-eabi,-D,__FPU_PRESENT,-D,MBED_CONF_PLATFORM_CRASH_CAPTURE_ENABLED>
```

Comment:
I am also not sure we are supposed to have `-D,__FPU_PRESENT,-D,MBED_CONF_PLATFORM_CRASH_CAPTURE_ENABLED` here. What do you think?